packages/core/src/shared/lsp/lspResolver.ts

```diff
@@ -87,19 +84,37 @@ export class LanguageServerResolver {
         }
     }
 
+    /**
+     * Show a toast notification with progress bar for lsp remote downlaod
+     * Returns a timeout to be passed down into httpFetcher to handle user cancellation
+     */
+    private async showDownloadProgress() {
+        const timeout = new Timeout(5000)
```

Comment:
probably want that 5000 store as a const somewhere at the top of the file so its easy to find